core/sql/optimizer/NAClusterInfo.cpp

```diff
@@ -350,14 +244,8 @@ NAClusterInfo::NAClusterInfo(CollHeap * heap)
         if ((nodeInfo[i].type & MS_Mon_ZoneType_Aggregation) != 0 ||
```

Comment:
From what I remember from the old times, edge nodes were not meant for ESPs, those should all go to aggregation nodes.